vortex-buffer/src/bool/bool.rs

```diff
@@ -0,0 +1,161 @@
+use crate::{buffer, Buffer};
+
+/// BoolBuffer provides a [`Buffer`] which encodes a packed bitset of boolean values.
+pub struct BoolBuffer {
+    buffer: Buffer<u64>,
```

Comment:
I know this is nice, but I think it prevents us from zero-copy conversion from Arrow BooleanBuffers, since they hold a ByteBuffer internally which may not have sufficient alignment for a u64

Reply:
Good point